### PR TITLE
doc: 16.2.5 Release Notes

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/releases/pacific.rst [hunk 1]
```diff
@@ -5,6 +5,374 @@ Pacific
 Pacific is the 16th stable release of Ceph.  It is named after the
 giant pacific octopus (Enteroctopus dofleini).
 
+v16.2.5 Pacific
+===============
+
+This is a hotfix release addressing a number of security issues and regressions. We recommend all users update to this release.
```
Comment (idryomov) [inline doc/releases/pacific.rst:11]: IIUC this is a regular backport release, not a hotfix one.

**Review by idryomov — CHANGES_REQUESTED** [review]
The changelog needs considerable massaging: lots of lines are wrapped, "refs/pull/41182/head"-like entries that shouldn't be there, etc.

**Review by sebastian-philipp — APPROVED** [review]

File: doc/releases/pacific.rst [hunk 2]
```diff
@@ -5,6 +5,182 @@ Pacific
...
+v16.2.5 Pacific
+===============
+
+This is the fifth  backport release in the Pacific series.  We recommend all users
+update to this release.
+
+Notable Changes
+--------------
+* .github/labeler: add api-change label (`pr#41818 <https://github.com/ceph/ceph/pull/41818>`_, Ernesto Pu
+erta)
```
Comment (idryomov) [inline doc/releases/pacific.rst:17]: This and many other lines are still wrapped.  It should be one physical line per change like in the second half of the changelog -- all wrapped lines seem to be in the first half for some reason.

File: doc/releases/pacific.rst [hunk 3]
```diff
@@ -5,6 +5,182 @@ Pacific
...
 giant pacific octopus (Enteroctopus dofleini).
 
+v16.2.5 Pacific
+===============
+
+This is the fifth  backport release in the Pacific series.  We recommend all users
+update to this release.
+
+Notable Changes
+--------------
```
Comment (idryomov) [inline doc/releases/pacific.rst:15]: Missing a dash or two here.

**Review by idryomov — CHANGES_REQUESTED** [review]

File: doc/releases/pacific.rst (hunk 3 above)
Comment (yuriw) [reply doc/releases/pacific.rst:15]: @idryomov I don't know why :(

@djgalloway  can you take a look ?  does the output need lot of editing after the script ran ?

Comment (djgalloway) [reply doc/releases/pacific.rst:15]: Yes.  The missing dash in the header is a manual process though.  Unrelated to `ceph-release-notes` output.

**Review by neha-ojha — COMMENTED** [review]
@yuriw a diff between v16.2.4..upstream/pacific shows a lot of content in `PendingReleaseNotes`, which I think should be included in the notable changes section. 
```
+
+* `ceph-mgr-modules-core` debian package does not recommend `ceph-mgr-rook`
+  anymore. As the latter depends on `python3-numpy` which cannot be imported in
+  different Python sub-interpreters multi-times if the version of
+  `python3-numpy` is older than 1.19. Since `apt-get` installs the `Recommends`
+  packages by default, `ceph-mgr-rook` was always installed along with
+  `ceph-mgr` debian package as an indirect dependency. If your workflow depends
+  on this behavior, you might want to install `ceph-mgr-rook` separately.
+
+* mgr/nfs: ``nfs`` module is moved out of volumes plugin. Prior using the
+  ``ceph nfs`` commands, ``nfs`` mgr module must be enabled.
+
+* volumes/nfs: The ``cephfs`` cluster type has been removed from the
+  ``nfs cluster create`` subcommand. Clusters deployed by cephadm can
+  support an NFS export of both ``rgw`` and ``cephfs`` from a single
+  NFS cluster instance.
+
+* The ``nfs cluster update`` command has been removed.  You can modify
+  the placement of an existing NFS service (and/or its associated
+  ingress service) using ``orch ls --export`` and ``orch apply -i
+  ...``.
+
+* The ``orch apply nfs`` command no longer requires a pool or
+  namespace argument. We strongly encourage users to use the defaults
+  so that the ``nfs cluster ls`` and related commands will work
+  properly.
+
+* The ``nfs cluster delete`` and ``nfs export delete`` commands are
+  deprecated and will be removed in a future release.  Please use
+  ``nfs cluster rm`` and ``nfs export rm`` instead.
```

On a separate note, we need to clean up the`PendingReleaseNotes` file for pacific.

**Comment by djgalloway** [comment]
Instead of asking @yuriw to do the fixes, squash commits, etc., I just created a separate PR: https://github.com/ceph/ceph/pull/42246

**Pull request closed without merging**